### PR TITLE
2017-detroit event created

### PR DESCRIPTION
**Pull request by bchess**
2017 Detroit is getting ready early! We have a contract signed for Sep 27 & 28.

I'm not sure if the organizers email list is created yet - you guys handle that right?

### CODE REVIEW

**Comment by MikeRosTX** [comment]
@bchess you have welcome  sponsor  contact  conduct  menu at the top, but it's missing location menu. Add location so I can  merge it.  Ping me a Slack if you need clarification.

File: data/events/2017-detroit.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+coordinates: "42.331034, -83.046382" # The corrodinates of your venue. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "A. Alfred Taubman Center for Design Education, College for Creative Studies" # The name of your location
+#
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: welcome
+#  - name: program
+#  - name: propose
+#    url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+#  - name: location
```
Comment (bridgetkromhout) [inline data/events/2017-detroit.yml]: Specifically, uncomment this if you want location linked in the nav.

**Comment by bridgetkromhout** [comment]
@MikeRosTX the location is linked in the body of the welcome page, so even though it's not in the nav I don't see that as a blocker to merging. (I clarified what they should do if they want it in the nav.)

**Comment by bridgetkromhout** [comment]
> 2017 Detroit is getting ready early! We have a contract signed for Sep 27 & 28.

@bchess exciting!

> I'm not sure if the organizers email list is created yet - you guys handle that right?

Yes. Since you didn't add any new people, I set the returning organizers up with their same emails as 2016 on the new mail aliases. I also disabled Slack access for the non-returning organizers.

**Comment by bchess** [comment]
Ah yeah, thanks for catching the location discrepancy. I'll fix that in a followup PR

**Pull request merged**